### PR TITLE
Feature/ivory 334 before you start page

### PR DESCRIPTION
**Pull request by Jozzey**
Created new page

### CODE REVIEW

File: server/views/can-continue.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+      <h1 class="govuk-heading-l" id="pageTitle">{{ pageTitle }}</h1>
+
+        <p class="govuk-body">You’ll need to:</p>
+
+        <ul class="govuk-list govuk-list--bullet">
+          <li>describe the item and how it meets the exemption criteria</li>
+          <li>upload some photos of it</li>
+          <li>provide contact details</li>
+          {{ unorderedList | safe}}
```
Comment (duncannixon) [inline server/views/can-continue.html:19]: Appreciate what you are doing here but I'm not too keen on the way the HTML for the unordered list items appears in the  .JS file, it feels like a breakage of the 'separation of concerns'.

I think a better way to achieve what you want here is to pass in the 1 or 2 items as a string array, then use the HTML below to render the HTML element(s) based on an 'additionalSteps' string array:

```
<ul....>
      <li>describe the item and how it meets the exemption criteria</li>
      <li>upload some photos of it</li>
      <li>provide contact details</li>

      {% for additionalStep in additionalSteps %}
        <li id="additionalStep-{{ loop.index }}">{{ additionalStep }}</li>
      {% endfor %}
</ul>
```

you will then have IDs of additionalStep-1 and possibly aadditionalStep-2 depending on the items passed in the array (i.e. a 1-based indexing)

**Review by duncannixon — APPROVED** [review]
I've made a recommendation regarding the dynamic list items, but other than that all good

**Review by duncannixon — CHANGES_REQUESTED** [review]
Oops approved by accident :)

File: server/views/can-continue.html (hunk 1 above)
Comment (Jozzey) [reply server/views/can-continue.html:19]: Thanks Duncan. Can't say I was that keen on what I did myself. The way you've suggested looks like a better way of doing it so I'll get my code updated.

**Comment by Jozzey** [comment]
Updated as per Duncan's comment

File: server/views/can-continue.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+
+{% block formContent %}
+  <div class="govuk-grid-row">
+
+    <div class="govuk-grid-column-two-thirds">
+
+      <h1 class="govuk-heading-l" id="pageTitle">{{ pageTitle }}</h1>
+
+        <p class="govuk-body">You’ll need to:</p>
```
Comment (duncannixon) [inline server/views/can-continue.html:13]: Add ID and unit test

File: server/views/can-continue.html [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+        <ul class="govuk-list govuk-list--bullet">
+          <li>describe the item and how it meets the exemption criteria</li>
+          <li>upload some photos of it</li>
+          <li>provide contact details</li>
```
Comment (duncannixon) [inline server/views/can-continue.html:18]: Add ID and unit tests for the 3 LI's

**Review by duncannixon — CHANGES_REQUESTED** [review]
Sorry just noticed that there are some HTML elements that don't have IDs. Also, ideally should have 1 or more unit tests that checks these elements- even though they are static text there is the theoretical risk that they could be modified or removed accidentally.

**Comment by Jozzey** [comment]
Added unit tests to static HTML as per Duncan's comment. Also added a test to the "Cancel" link

**Review by duncannixon — APPROVED** [review]
Good work

**Pull request merged**